sdk/core/src/lib.rs

```diff
@@ -57,6 +57,13 @@ pub use response::*;
 pub use seekable_stream::*;
 pub use sleep::sleep;
 
+// re-export packages
+pub use http_types as http;
+pub use url;
+
+// re-export important types at crate level
+pub use http_types::Method;
+pub use http_types::StatusCode;
 pub use url::Url;
```

Comment:
I like the idea of having important types at the crate level like `Request` and `Response`:
- Url
- Method
- StatusCode